packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -881,22 +879,24 @@ export const startStakeFactory = async (
   const storageNode = await makeStorageNode(chainStorage, STORAGE_PATH);
   const marshaller = await E(board).getReadonlyMarshaller();
 
-  const stakeTerms = await deeplyFulfillTerms({
-    timer,
-    electorateInstance,
-    governedContractInstallation: installations.stakeFactory,
-    governed: harden({
-      terms: stakeFactoryTerms,
-      issuerKeywordRecord: { Stake: bldIssuer },
-      privateArgs: {
-        feeMintAccess,
-        initialPoserInvitation,
-        lienBridge,
-        storageNode,
-        marshaller,
-      },
+  const stakeTerms = await deeplyFulfilled(
+    harden({
+      timer,
+      electorateInstance,
+      governedContractInstallation: installations.stakeFactory,
+      governed: harden({
+        terms: stakeFactoryTerms,
+        issuerKeywordRecord: { Stake: bldIssuer },
+        privateArgs: {
+          feeMintAccess,
+          initialPoserInvitation,
+          lienBridge,
+          storageNode,
+          marshaller,
```

Comment:
Re line 890 above

Terms are not private. Perhaps I am missing context, but it seems wrong that something called somethingTerms contains something called privateArgs. Doesn't this cause the allegedly private args to be exposed to anyone with access to these terms?